include/ck/tensor_operation/gpu/grid/gridwise_sparse_embeddings_forward_layernorm.hpp

```diff
@@ -205,14 +173,10 @@ struct GridwiseSparseEmbedding3ForwardLayernorm
                 static_for<0, RowVectorSize, 1>{}([&](auto i_row_vec_) {
                     constexpr auto register_offset = thread_buf_desc.CalculateOffset(
                         make_tuple(i_dim_sub_, i_dim_vec_, i_row_sub_, i_row_vec_));
-                    AccDataType va =
-                        ck::type_convert<AccDataType>(in_thread_buf_a(Number<register_offset>{}));
-                    AccDataType vb =
-                        ck::type_convert<AccDataType>(in_thread_buf_b(Number<register_offset>{}));
-                    AccDataType vc =
-                        ck::type_convert<AccDataType>(in_thread_buf_c(Number<register_offset>{}));
-
-                    acc_thread_buf(Number<register_offset>{}) += va + vb + vc;
+                    static_for<0, NumEmbeddings, 1>{}([&](auto i_embedding_) {
+                        acc_thread_buf(Number<register_offset>{}) += ck::type_convert<AccDataType>(
```

Comment:
We hardcode the elementwise add after embedding in this kernel.
Hence, it is better to renamed the kernel as embedding_add_layernorm_forward or pass the lambda function (emb_elementwise).

Reply:
Done